pyqtgraph/graphicsItems/tests/test_PlotCurveItem.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import pyqtgraph as pg
+from pyqtgraph.tests import assertImageApproved
+
+
+def test_PlotCurveItem():
+    p = pg.plot()
+    p.resize(200, 150)
+    data = np.array([1,4,2,3,np.inf,5,7,6,-np.inf,8,10,9,np.nan,-1,-2,0])
```

Comment:
Even better catch. I made the image comparison more strict; seems to be working now.